include/translatable-object.php

```diff
@@ -379,17 +379,35 @@ public function get_objects_with_no_lang( $limit, array $args = array() ) {
 			return array();
 		}
 
+		$object_ids = $this->query_objects_with_no_lang( $sql );
+
+		return array_values( $this->sanitize_int_ids_list( $object_ids ) );
+	}
+
+	/**
+	 * Returns object IDs without language given a specific SQL query.
+	 * Can be overriden by child classes in case queried object doesn't use
+	 * `wp_cache_set_last_changed()` or another cache system.
+	 *
+	 * @since 3.4
+	 *
+	 * @param string $sql A prepared SQL query for object IDs with no language.
+	 * @return mixed An array of numeric object IDs.
```

Comment:
Since `wpdb::get_col()` returns an array, I think it's safe to use a more specific return type:
```suggestion
	 * @return string[] An array of numeric object IDs.
```